### PR TITLE
Nice

### PR DESCRIPTION
**Pull request by osherboudara99**

### CODE REVIEW

**Comment by eytan-ohana** [comment]
This is my comment on your commit. make another

**Comment by osherboudara99** [comment]
hey bro

File: file.txt [hunk 1]
```diff
@@ -1 +1,4 @@
-Dip slip lalalalalalalal another cha g
\ No newline at end of file
+Final
+
+
+final2
```
Comment (eytan-ohana) [inline file.txt:4]: put a space between final and 2

Comment (eytan-ohana) [reply file.txt:4]: @osherboudara99

**Pull request merged**